### DISCUSSION TITLE
PsApi GetProcessMemoryInfo

### DISCUSSION THREAD

**Discussion by lucas-wheeler** [microsoft/win32metadata#526] [Q&A] [answered]
PsAPI only four functions implemented, so i tried to generate GetProcessMemoryInfo which i need. however, i get the error: NativeMethods.txt(1,1,1,8): warning PInvoke001: No methods found under module "Psapi".

i have created a console app that just has CsWin32 installed, through nuget
`<PackageReference Include="Microsoft.Windows.CsWin32" Version="0.1.422-beta">`

my NativeMethods.txt has:
Psapi.*

is there something that i'm missing?

**Comment by AArnott** [comment, marked as answer]
The metadata doesn't appear to have the functions you're looking for. 
But interestingly, I see `K32GetProcessMemoryInfo` defined, but that comes from Kernel32. That's actually [consistent with documentation](https://docs.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-getprocessmemoryinfo) except the method name doesn't match. What's going on here, @sotteson1?

**Reply by mikebattista** [reply]
This was fixed with https://github.com/microsoft/win32metadata/issues/920.
